CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Add `--profile=PERFORMANCE` for high spec nodes: increases rocksdb cache and enables parallel transaction processing [#8560](https://github.com/hyperledger/besu/pull/8560)
 - Add `--profile=PERFORMANCE_RPC` for high spec RPC nodes: increases rocksdb cache and caches last 2048 blocks [#8560](https://github.com/hyperledger/besu/pull/8560)
 - Refine gas estimation algorithm for `eth_estimateGas` and `eth_createAccessList` [#8478](https://github.com/hyperledger/besu/pull/8478) including a new option to specify `--estimate-gas-tolerance-ratio`
+- Default target-gas-limit to 60M for sepolia and other testnets [#8594](https://github.com/hyperledger/besu/pull/8594)
```

Comment:
Think we should be explicit about the testnets, i.e. "hoodi, holesky, ephemery". Or just sepolia if we go with that option